source/_docs/scripts/service-calls.markdown

```diff
@@ -176,6 +176,87 @@ data:
 
 {% endraw %}
 
+### Use response data in templates: convert YAML to JSON
+
+The response data is returned in Yaml, and can not be used as is in the template editor.
+The response needs to be converted to JSON to be able to do so.
```

Comment:
Why?

Reply:
believe this is the same question?
I merely tried to describe the full process of getting the response, and handling that response to finally get some format we can use in the templates.

as I see it, we need to describe the difference between the data in yaml format, and how to make  a json object of it to be used in templates

its not just some additional format, it is what Home Assistant returns, there is no way around that